src/main/java/net/glowstone/GlowServer.java

```diff
@@ -425,6 +434,13 @@
      * Whether the macOS/BSD kqueue native transport is available for Netty.
      */
     public static final boolean KQUEUE = KQueue.isAvailable();
+    /**
+     * Libraries that are known to cause problems when loaded up via the server config or plugin
+     * dependencies.
+     */
+    private static final Set<LibraryKey> blacklistedRuntimeLibs = ImmutableSet.of(
+            new LibraryKey("it.unimi.dsi", "fastutil")
```

Comment:
Why is this blacklisted? What problems does it cause?

Reply:
It causes classpath conflicts with FastUtil-Lite, which we include in the server JAR.